fastcrypto-vdf/src/vdf/mod.rs

```diff
@@ -38,29 +38,24 @@ pub trait VDF {
 mod tests {
     use crate::class_group::discriminant::Discriminant;
     use crate::class_group::QuadraticForm;
-    use crate::vdf::wesolowski::StrongVDF;
+    use crate::vdf::wesolowski::DefaultVDF;
     use crate::vdf::VDF;
-    use crate::ToBytes;
     use fastcrypto::hash::{HashFunction, Sha256};
     use num_bigint::BigInt;
-    use num_traits::Num;
+    use std::str::FromStr;
 
     #[test]
-    fn vdf_e2e_test_1024() {
-        // This test runs an e2e test of the VDF protocol as it is supposed to be run on SUI with a smaller discriminant
-        // and a smaller number of iterations.
+    fn vdf_e2e_test() {
+        // This test runs an e2e test of the VDF-based randomness protocol with a 3072 bit discriminant.
 
-        // Fixed 1024 bit discriminant. In production this should be bigger.
-        let discriminant_bytes = "fdf4aa9b7f49b85fc71f6fbf31a3d51e6828afb9d06165f5814bb5142485853abb52f50b7c8a937bba09ce75b51a639886d997d561b7a654f1a9e6b66645d76fad093381d464eccf28d599fb5a938bb99101c30e374f5f786c9232f56d0118826d113400b080bb4737018b088af5203a18da25d106fffdad7e8f660e141dd11f";
+        // Fixed 3072 bit discriminant.
```

Comment:
how did you generate that?

Reply:
Using a trivial [1,2,3] seed. I'll add a comment.